config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
   # You can have the root of your site routed with "root"
   root 'welcome#index'
-
+  get 'game_day', :to => 'game_day#index', :as => 'game_day'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.